runtimes/adapters/internal_agent.py

```
@@ -38,7 +38,31 @@
+    """
+    try:
+        from packages.ai.brain_config import (
+            PROVIDER_CANDIDATES,
+            SAFE_DEFAULT_MODEL,
+        )
+        cands = PROVIDER_CANDIDATES.get("nvidia") or []
+        # First candidate is the catalog preset for nvidia's executor role.
+        if cands:
+            return cands[0]
+        return SAFE_DEFAULT_MODEL
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)